easybuild/easyconfigs/h/h5py/h5py-3.12.1-foss-2024a.eb

```
@@ -0,0 +1,39 @@
+easyblock = 'PythonPackage'
+
+name = 'h5py'
+version = '3.12.1'
+
+homepage = 'https://www.h5py.org/'
+description = """HDF5 for Python (h5py) is a general-purpose Python interface to the Hierarchical Data Format library,
+ version 5. HDF5 is a versatile, mature scientific software library designed for the fast, flexible storage of enormous
+ amounts of data."""
+
+toolchain = {'name': 'foss', 'version': '2024a'}
+
+toolchainopts = {'usempi': True}
+
+sources = [SOURCE_TAR_GZ]
+checksums = ['326d70b53d31baa61f00b8aa5f95c2fcb9621a3ee8365d770c551a13dbbcbfdf']
+
+builddependencies = [
+    ('pkgconf', '2.2.0'),
+    ('Cython', '3.0.10'),
+]
+
+dependencies = [
+    ('Python', '3.12.3'),
+    ('SciPy-bundle', '2024.05'),
+    ('mpi4py', '4.0.1'),
+    ('HDF5', '1.14.6', '', {'name': 'gompi', 'version': '2024a'}),
```

```suggestion
    ('HDF5', '1.14.5'),
```
The version already in EasyBuild is 1.14.5 (https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/h/HDF5/HDF5-1.14.5-gompi-2024a.eb)

---

I see I got `HDF5-1.14.6-gompi-2024a.eb` locally. Should I also commit that in a separate PR, or should I just go with v1.14.5 as you suggested above?